lib/ChainHandler.js

```diff
@@ -0,0 +1,46 @@
+/* @flow weak */
+"use strict";
+
+var async = require("async");
+
+var ChainHandler = module.exports = function ChainHandler() { };
+
+ChainHandler.prototype.chain = function(otherHandler) {
```

Comment:
I like the use of `chain` as when we start stacking up these handlers it won't be very obvious where the magic is coming from - by including a part of the name in the function we're helping to steer newcomers towards the right documentation.